tests/custom_lookups/tests.py

```diff
@@ -249,6 +249,28 @@ def test_custom_name_lookup(self):
             self.assertSequenceEqual(qs1, [a1])
             self.assertSequenceEqual(qs2, [a1])
 
+    def test_custom_lookup_with_subquery(self):
+        class NotEqual(models.Lookup):
+            lookup_name = "ne"
+
+            def as_sql(self, compiler, connection):
+                lhs, lhs_params = self.process_lhs(compiler, connection)
+                rhs, rhs_params = self.process_rhs(compiler, connection)
+                params = lhs_params + rhs_params
```

Comment:
Did you use the `a + b` style over the `(*a, *b)` intentionally here? Seems inconsistent with the other tests and the docs...

Reply:
Great question. Yes, I did this intentionally to capture the scenario that failed with `TypeError`. If I adjust this test to the more resilient pattern with unpacking, I won't capture the issue.

Do you think adding a comment would help clarify?

> Seems inconsistent with the other tests and the docs...

Re the docs, I've only documented the unpacking syntax in the release note (addressed to users supporting multiple versions of Django) since part of the point of this change is that the prior documented behavior (which was working in most cases) now works in the edge case with `Subquery` on some databases. So I wouldn't call it inconsistent with the docs.

Reply:
Thanks for having a look!

Reply:
Ah got you. Thanks for the explanation. 

> Do you think adding a comment would help clarify?

Perhaps... I stumbled on this issue in the review queue and had some basic context, so I can imagine someone else looking at the test in the future might think the same as I did? I might have been biased by the surrounding changes in the diff.

Reply:
Pushed a little comment in 6f1ada81c34df69290d80fa739c5956a04573bb3. Noticed there are merge conflicts, so I'll rebase in a sec.

Reply:
And if you're still at the sprints, raise your hand or come say hello!

Reply:
I'm around today, yes 😄